BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/DigitalInvoice/Model/DigitalInvoice.swift

```diff
@@ -14,13 +14,16 @@ import GiniBankAPILibrary
  */
 public struct DigitalInvoice {
 
-    private let _extractionResult: ExtractionResult
+    let _extractionResult: ExtractionResult
```

Comment:
[clarification] why we change this from being private?

Reply:
I found an option how to keep it private